setup/install_upstream_rust_keylime/test.sh

```diff
@@ -59,6 +75,29 @@ _EOF'
 [Service]
 Environment=\"RUST_LOG=keylime_agent=trace\"
 _EOF"
+            if [ -n "$KEYLIME_RUST_CODE_COVERAGE" ]; then
```

Comment:
same as above. Maybe process that variable at the beginning of the script like
```
if [ "$KEYLIME_RUST_CODE_COVERAGE" == "1" -o "$KEYLIME_RUST_CODE_COVERAGE" == "true" ]; then
  MEASURE_COVERAGE=true
else
  MEASURE_COVERAGE=false
fi
```
and later use just `if $MEASURE_COVERAGE; then ...`

Reply:
Why we would use another variable, when we can you variable already defined in plan? @kkaarreell

Reply:
So I used the variable from plan if don't mind.